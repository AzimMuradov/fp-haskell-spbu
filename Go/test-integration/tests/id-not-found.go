// should fail with "panic: analyzer error: identifier b not found"

func main() {
  var a int;

  {
    println(a); // should print nothing (because of analyzer failure)
  };

  println(a);   // should print nothing (because of analyzer failure)
  println(b);   // should print nothing (because of analyzer failure)
}
