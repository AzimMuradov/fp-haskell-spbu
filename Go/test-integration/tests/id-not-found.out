panic: analysis error: identifier b not found
